wp-content/themes/delta-child/single-projects.php

```diff
@@ -213,16 +213,31 @@ function include_media_slider() {
                                 $url = get_permalink($media_id);
                                 ?>
                                 <div class="swiper-slide" style="height: 500px; width: auto;">
-                                <?php 
+                                <?php
                                     if (wp_attachment_is( 'image', $media_id)) {
-                                        ?> 
+                                        ?>
                                          <img src="<?php echo $url ?>"></img>
                                         <?php
                                     }
 
-                                    if (wp_attachment_is( 'video', $media_id)) {
+                                    if (wp_attachment_is('video', $media_id)) {
                                         ?>
-                                            <video controls src="<?php echo $url ?>"></video>
+                                        <div class="video-container" style="position: relative;">
+                                            <video class="video-element" src="<?php echo $url ?>" style="width: 100%; height: 100%; pointer-events: none;""></video>
+                                            <div class="play-button" style="position: absolute; top: 50%; left: 50%; transform: translate(-50%, -50%); z-index: 10; cursor: pointer;">
+                                                <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="64" height="64" style="fill: #FFFFFF;">
+                                                    <path d="M8 5v14l11-7z"/>
+                                                    <path d="M0 0h24v24H0z" fill="none"/>
+                                                </svg>
+                                            </div>
+                                            <div class="pause-button" style="display: none; position: absolute; top: 50%; left: 50%; transform: translate(-50%, -50%); z-index: 10; cursor: pointer;">
+                                                <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="64" height="64" style="fill: #FFFFFF;">
+                                                    <path d="M6 19h4V5H6v14zm8-14v14h4V5h-4z"/>
+                                                    <path d="M0 0h24v24H0z" fill="none"/>
+                                                </svg>
+                                            </div>
+
+                                        </div>
```

Comment:
Controls work great, however on a larger screen the video size outgrows the container. This should stay the same size as the images since these sizes are handled by Swiperjs. Seems to happen because of the video wrapper.

This image displays the issue.
![image](https://github.com/xl3ehindTim/delta-site/assets/32740573/35d72bd0-3473-4edf-b0c8-b4d8aa896b91)